### ISSUE TITLE
Log Name Issues - FQDN Limitation

### ISSUE THREAD

**Issue by jcdmacleod** [Packet-Clearing-House/DNSAuth#29]
DNSAuth doesn’t accept a file unless it is:

mon-01.pop.example.net_2018-12-29.05-59.dmp

This fails:

mon-01.pop.example.com_2018-12-29.05-59.dmp

I have not tried other tlds, but .com certainly fails.  The files are just ignored, a rename causes them to be ingested instantly.

I will posted this under PDP too.

**Comment by mrjones-plip** [comment]
@unix1  - when you're next in the DNSAuth code, can you confirm we actually only accept the ``.net`` TLD in the FQDN?  We can triage this ticket accordingly after that.

**Comment by unix1** [comment]
@Ths2-9Y-LqJt6 that is correct. The `aggregate` function has `net_` hardcoded in file name parsing and it expects it there.

**Comment by mrjones-plip** [comment]
@unix1  - great, thanks for the confirmation. If this is as easy of a fix as i think it is, please grab when you're next doing a cycle of work.  No rush. 

Conversely, if it's hard, or will have possible unknown reproductions, let's hold off until this can be the primary ticket you're working on.

@jcdmacleod  - thanks for your patience while we work through this!

**Comment by unix1** [comment]
@Ths2-9Y-LqJt6 this should be very simple indeed.
